src/store/reducers/lessonsReducer.js

```diff
@@ -14,6 +17,11 @@ const initState = {
 
 const reducer = (state = initState, action) => {
   switch (action.type) {
+    case CREATE_LESSONS_REQUEST:
+      return { ...state, loading: true };
+    case CREATE_LESSONS_SUCCESS:
+      return { ...state, loading: false };
+    case CREATE_LESSONS_FAILURE:
```

Comment:
```suggestion
    case CREATE_LESSONS_FAILURE:
```
стэйт не будет менять при ошибке?

Reply:
исправлено